test/e2e/examples.go

```diff
@@ -167,6 +167,51 @@ var _ = Describe("Examples e2e", func() {
 			})
 		})
 	})
+
+	Describe("[Skipped][Example]Storm", func() {
```

Comment:
IIRC `[Skipped]` is a syntax specifically for density.go, right?  Or is it broader now.